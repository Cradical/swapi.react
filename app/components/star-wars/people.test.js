import React from 'react'
import StarWarsPeople from './people'
import renderer from 'react-test-renderer'

test('Star Wars People', () => {
  const component = renderer.create(
    <StarWarsPeople />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
