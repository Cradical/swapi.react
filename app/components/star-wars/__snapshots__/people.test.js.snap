// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Star Wars People 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <h1>
    Star Wars People
  </h1>
  <ul />
</div>
`;
